lib/docker/container.go

```diff
@@ -35,7 +37,15 @@ func CreateContainer(containerCfg *types.ApplicationContainer) (string, error) {
 		Volumes: map[string]struct{}{
 			volume: {},
 		},
+		Healthcheck : &container.HealthConfig{
+			Test: []string{"CMD-SHELL", fmt.Sprintf("curl --fail http://localhost:%d/ || exit 1", containerCfg.ContainerPort)},
```

Comment:
change `containerCfg.ContainerPort` to `containerCfg.ApplicationPort` and add the `--silent` flag to the curl command

Reply:
@Scar26 you forgot these changes

Reply:
sorry, I accidentally reverted that commit as well